src/js/map/Pathing.js

```diff
@@ -24,13 +24,41 @@ export default class Pathing {
     this.compute()
   }
 
+  reset() {
+    this.weights.reset()
+    // this.compute()
```

Comment:
Is there a reason we don't recompute the pathing map now that the weights are empty?

Reply:
Good point! I think this was something that I toggled while trying to identify the source of the problem. It's been changed back now.